### PR TITLE
Enables respawns in default config

### PR DESCRIPTION
**Pull request by iloveyoulots**
#232  About The Pull Request

Changes the default config to have respawns enabled. 

## Why It's Good For The Game

Respawns are an important feature most 'long round' furry servers allow for, and allow people to change characters during the round. The majority of other large servers such as this allow respawns by default now, this is just a change to bring us closer to the standard others have.

## Changelog
:cl:
config: Enables respawns by default in the config.
/:cl:

### CODE REVIEW

**Comment by Dahlular** [comment]
Right, this goes against a lot of what hyper stands for. Respawns are very finnicky with people and can enable those to infinitely validhunt/shitter about. I'm not completely opposed to respawns entirely though; if we wanted respawns, a system where people can respawn as a different character (perhaps with a 30 minute timer) would be excellent. I've had discussions about this in the past, and that's not too bad of an idea.
On top of that, perhaps "respawning periods" can be implemented as well. Such as, whenever a crew transfer vote fails and the round continues, dead players can respawn as a different character directly into the round. My main concern here is people joining as the same character so quickly; I'd much prefer a system where people can respawn in a given amount of time as another character in their roster.

TL;DR, respawns are not going to be something that will just be handed out so easily.
